docs/data-sources/dcs_flavors.md

```diff
@@ -0,0 +1,77 @@
+---
+subcategory: "Distributed Cache Service"
+---
+
+# huaweicloud_dcs_flavors
+
+Use this data source to get a list of available DCS flavors.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_dcs_flavors" "flavors" {
+  engine   = "Redis"
+  capacity = "4"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) The region in which to obtain the DCS flavors.
+  If omitted, the provider-level region will be used.
+
+* `engine` - (Required, String) The engine of the cache instance. Valid values are *Redis* and *Memcached*.
+
+* `capacity` - (Required, String) The cache capacity. Unit: GB.
+  + **Redis4.0 and Redis5.0**: Stand-alone and active/standby type instance values:
+    0.125, 0.25, 0.5, 1, 2, 4, 8, 16, 32, 64.
+    Cluster instance specifications support 24, 32, 48, 64, 96, 128, 192, 256, 384, 512, 768, 1024.
```

Comment:
Shall we need to use ** or ` to mark numbers ?

Reply:
1. `1024 ` is an input parameter,  changed to *, **,  no data can be matched.
2. Because the subtitle is bolded. I think it’s easier to read in the format \`xxx\`.

Reply:
I means that number should be separated into thousands with commas.